README.md

```diff
@@ -178,6 +178,74 @@ It is possible to retrieve the audit trail of an use case after its execution
     ]
 }
 ```
+You can also configure the output to ignore some fields of the audit trail by choosing those options (mode or output):
+
+```javascript
+    auditTrail: {
+        mode: "minimal", // if minimal won't output especific fields: steps, request, return and user
+        output:{
```

Comment:
I put output precisely to make it more evident what those true or false variables would be. Precisely so that there are no breaks in future developments.